### ISSUE TITLE
Adding URL dynamically reset the index at 0

### ISSUE THREAD

**Issue by MarcARoberge** [dockleryxk/ng-simple-slideshow#87]
I am pushing new photos urls inside my array but it keeps resetting the slideshow to the index 0.
```
 this.photoService.getPhotoURLWithCountryName(this.country.name, this.currentIndex, 1)
          .then((photosData: string[]) => {
            this.photosURLs.push(photosData[0]);
          })
          .catch(e => console.error(e));
```

Ps: Sorry for formatting my issue wrongly. I am not used to publish on GitHub

**Comment by dockleryxk** [comment]
@MarcARoberge try this instead 

```
this.photoService.getPhotoURLWithCountryName(this.country.name, this.currentIndex, 1)
          .then((photosData: string[]) => {
            this.photosURLs.push(...photosData[0]);
          })
          .catch(e => console.error(e));
```

**Comment by dockleryxk** [comment]
@MarcARoberge did this work for you? Going to close this unless I hear back

**Comment by MarcARoberge** [comment]
Sorry never got the email of you answering! I am going to check thank you !

**Comment by MarcARoberge** [comment]
@dockleryxk So since I am using base64 encode picture (I am retrieving files from my computer) copying the object  is taking way too long. I am going to create a PR. I fixed it by modifying the module a little bit to not reset to index when when the array is filling.

**Comment by dockleryxk** [comment]
@MarcARoberge as long as it still works correctly, I am all for it!

**Comment by MarcARoberge** [comment]
#89 :)

**Comment by dockleryxk** [comment]
Available on 1.3.0-beta.3
